doc/source/reference/c-api/types-and-structures.rst

```diff
@@ -1429,7 +1429,7 @@ for completeness and assistance in understanding the code.
    to define a 1-d loop for a ufunc for every defined signature of a
    user-defined data-type.
 
-.. c:type:: PyArrayMapIter_Type
+.. data:: PyArrayMapIter_Type
```

Comment:
```suggestion
.. c:var:: PyArrayMapIter_Type
```

Reply:
`.. c:var::` is equivalent to `.. c:member::` (in version 3), so cannot be used here. The error is:
```
WARNING: Error in declarator or parameters
Invalid C declaration: Expected identifier in nested name. [error at 16]
  PyArrayIter_Type
  ----------------^
```

Reply:
This sounds like a sphinx bug to me, I'd encourage you to report it upstream.

Reply:
I was misunderstanding about the error, and the cause was the lack of type specifications.